xbmc/video/VideoDatabase.h

```diff
@@ -797,6 +797,11 @@ class CVideoDatabase : public CDatabase
 
   virtual bool GetFilter(CDbUrl &videoUrl, Filter &filter, SortDescription &sorting);
 
+  /*! \brief Will check if the season exists and if that is not the case add it to the database.
+  \param showID The id of the show in question.
+  \param season The season number we want to add.
+  \return The id of the season.
```

Comment:
Please change to dbId to have it consistent and to make it clear it's the value used for `m_iDbId`?